sigstore-java/src/main/java/dev/sigstore/KeylessSigner.java

```diff
@@ -123,9 +168,11 @@ public Builder sigstorePublicDefaults()
           RekorVerifier.newRekorVerifier(VerificationMaterial.Production.rekorPublicKey()));
       oidcClient(WebOidcClient.builder().build());
       signer(Signers.newEcdsaSigner());
+      minSigningCertificateLifetime(Duration.ofMinutes(5));
```

Comment:
I think it's a bit confusing to set the default in the builder and here? Either make it a const in the class or remove this?

Reply:
oh that's interesting, maybe `sigstorePublicDefaults` should be static on the KeylessSigner class rather than on the builder.